### PR TITLE
Bugfix: Handling Multiple Colons in PR Summary Title

### PR DESCRIPTION
**Pull request by saumilsdk**
## PR Type:
Bug fix

___
## PR Description:
This PR addresses a bug related to the handling of PR summary titles that contain multiple colons. The fix involves replacing multiple colons in the same line with a single colon to ensure proper parsing of the PR summary title.

___
## PR Main Files Walkthrough:
`pr_agent/algo/utils.py`: Added a loop to iterate over each line of the review text. If a line contains two or more colons, the line is partitioned at the first colon and all subsequent colons in the remaining part of the line are replaced with an empty string. This ensures that only the first colon is retained in each line.

### CODE REVIEW

File: pr_agent/algo/utils.py [hunk 1]
```diff
@@ -271,6 +271,13 @@ def load_yaml(review_text: str) -> dict:
 
 def try_fix_yaml(review_text: str) -> dict:
     review_text_lines = review_text.split('\n')
+
```
Comment (okotek) [inline pr_agent/algo/utils.py:274]: If the malformed YAML read something like:
title: My:title

A better approach could be to quote the value? Those colons may be significant..

**Comment by mrT23** [comment]
Preparing pr description...

**Comment by mrT23** [comment]
@saumilsdk 
https://github.com/Codium-ai/pr-agent/pull/200 should fix the issue by using block scalar format, and is more general.
You are welcome to test it

**Pull request closed without merging**